src/doc/config.md

```diff
@@ -95,3 +100,5 @@ Cargo recognizes a few global environment variables to configure how it runs:
   `rustdoc` instance instead.
 * `CARGO_TARGET_DIR` - Location of where to place all generated artifacts,
   relative to the current working directory.
+
+Settings specified via config files take precedence over those specified via environment variables.
```

Comment:
I just checked the behaviour of cargo, and the statement:

> Settings specified via config files take precedence over those specified via environment variables.

appears to be true.

As per https://github.com/rust-lang/cargo/blob/master/src/cargo/util/config.rs#L241, the config is checked first, and *then* the env variable is checked.

The current behaviour makes sense for the use case I desire:
I need to produce a self contained cross-compilation toolchain which can be distributed to other team members as part of a larger project. That project will have `.cargo/config` (with it's fancy relative path'd linker, ar and rustc tool specifiers) committed to git repo of that project. These developers might already have rust installed on their machines, but it'd likely be a major release of rust without cross-compilation support. As a result, they might have RUSTC already set within their environment variables, but it'd likely be pointing at their pre-existing rust. I want the project-level config to take precedence.

I can also see the other side of the argument, where it'd make more sense for env variable to take precedence.